### ISSUE TITLE
"npm init" results in "ERR! enoent This is related to npm not being able to find a file."

### ISSUE THREAD

**Issue by ptmkenny** [lando/lando#2211]
I'm getting an error when I try to use `npm`.

The error:

```
lando npm init
...just accept default values by pressing enter...
Is this OK? (yes)
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /app/web/themes/custom/mytheme/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/app/web/themes/custom/mytheme/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2020-05-04T08_50_24_078Z-debug.log

```

I don't know much about node so there may be something obvious I'm getting wrong.

I'm on Windows 10 using Powershell 7.

I tried running `lando rebuild`, and sometimes that allowed me to run a single command before the error returned.  But sometimes when I run `lando rebuild`, I still get the error.

lando.yml:

```
name: mysite
recipe: pantheon
config:
  xdebug: true
  framework: drupal8
  env: lando
  webroot: web

services:
  node:
    type: node:12
    port: 3000 # for browsersync
    globals:
      gulp-cli: latest
  database: # needed until pantheon recipe supports more recent mariadb
    # https://github.com/lando/lando/issues/2197
    type: mariadb:10.3
  appserver:
    overrides:
      environment:
        BEHAT_PARAMS: >-
          {"extensions" : {"Behat\\MinkExtension" : {"base_url" :
          "https://appserver_nginx/"}, "Drupal\\DrupalExtension" : {"drush" :   {
          "root":  "/app/web" }}}}

tooling:
  backstop:
    service: qa
    cmd:
      - "backstop reference --config=.ci/test/visual-regression/backstopConfig.js"
      - "backstop test --config=/app/.ci/test/visual-regression/backstopConfig.js"
  behat:
    service: appserver
    cmd: "/app/vendor/bin/behat -c /app/tests/behat/behat-lando.yml"
  npm:
    service: node
  node:
    service: node
  lighthouse:
    service: node

```

**Comment by pirog** [comment]
@ptmkenny i am not able to replicate this, can you provide any more details?

**Comment by ptmkenny** [comment]
Hmm, this was strange.  I rebooted my machine several times and the problem was persistent between reboots.

I uninstalled scoop, which I had used to install node in Windows, and the problem still occurred.

Then I realized that Powershell 7 and the built-in Powershell are different, so I opened the built-in powershell and ran `lando npm init` and it worked no matter how many times I did it.

So I went back to Powershell 7... and it worked there too.

In the end I wasn't able to find the true cause because it fixed itself mysteriously, but I will report back if it recurs and I figure out what's going on.
